### PR TITLE
📦 NEW: News app

### PR DESCRIPTION
**Pull request by msaaddev**
In this PR, I have included the source code of the `news-app` I built for an article. [Click here](https://rapidapi-example-news-app.vercel.app/) for a live preview.

@ghoshnirmalya kindly take a look.

### CODE REVIEW

**Comment by msaaddev** [comment]
@ghoshnirmalya I have deployed my forked repo. I don't know why Vercel is giving this error.

File: news-app/pages/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+export default function Home({ value }) {
+  return (
+    <div className="flex justify-center items-center flex-col">
+      <h3 className="text-secondary text-2xl font-raleway font-bold uppercase tracking-wide mb-10 md:text-lg">
+        Get Top <span className="text-danger">News</span> Quickly
+      </h3>
+      {value.map((news) => {
+        return (
+          <Link href={news.url}>
```
Comment (ghoshnirmalya) [inline news-app/pages/index.js:11]: There should be an anchor tag immediately after the Next.js `Link` component [unless the child is a functional component](https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-functional-component).

Comment (ghoshnirmalya) [reply news-app/pages/index.js:11]: For Lists or Arrays, you need to use [keys](https://reactjs.org/docs/lists-and-keys.html#keys).

File: news-app/tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  mode: "jit",
+  purge: ["./pages/**/*.{js,ts,jsx,tsx}", "./components/**/*.{js,ts,jsx,tsx}"],
+  darkMode: false, // or 'media' or 'class'
+  theme: {
+    screens: {
+      xl: { max: "1279px" },
```
Comment (ghoshnirmalya) [inline news-app/tailwind.config.js:7]: Do we need to override these for our application?

**Comment by msaaddev** [comment]
> > @ghoshnirmalya I have deployed my forked repo. I don't know why Vercel is giving this error.
> 
> Did you deploy on your own Vercel account?

Yes.

**Comment by msaaddev** [comment]
> @msaaddev You can ignore the issue with the deployment. Would you mind working on this PR and getting it merged today?

Already working on it. 🙌🏻

File: news-app/pages/api/news.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      url: "https://bing-news-search1.p.rapidapi.com/news",
+      params: { safeSearch: "Off", textFormat: "Raw" },
+      headers: {
+        "x-bingapis-sdk": "true",
+        "x-rapidapi-host": "bing-news-search1.p.rapidapi.com",
+        "x-rapidapi-key": process.env.NEXT_PUBLIC_BING_NEWS_API_KEY,
+      },
+    };
+
+    console.log();
```
Comment (ghoshnirmalya) [inline news-app/pages/api/news.js:16]:
```suggestion
```

**Review by ghoshnirmalya — CHANGES_REQUESTED** [review]

File: news-app/tailwind.config.js (hunk 2 above)
Comment (ghoshnirmalya) [reply news-app/tailwind.config.js:7]: This issue is still not resolved.

**Comment by msaaddev** [comment]
@ghoshnirmalya kindly take a look now.

**Review by ghoshnirmalya — APPROVED** [review]

**Pull request merged**